app/assets/javascripts/manage_submissions.js

```diff
@@ -45,17 +44,16 @@ jQuery(function($) {
         '<div class="row"><input type="checkbox" id="only-latest">' +
         '<label for="only-latest">Show only latest</label></div>'
     },
-    "columnDefs": [{
-      "targets": [7],
-      "visible": false,
-      // "searchable": false
-    }],
     "aaSorting": [
       [3, "desc"]
+    ],
+    "aoColumnDefs": [
+      { "bSortable": false, "aTargets": [ 0 ] },
+      {"bVisible": false, "aTargets": [7]}
     ]
   });
 
-  $("#only-latest").on("change", function() {
+  $("#only-latest").on("change", function() {;
```

Comment:
What is this semicolon for?

Reply:
ends that nonexistent line, its ok in JS random semicolons are a feature